compiler/p/codegen/OMRMemoryReference.cpp

```diff
@@ -1636,7 +1636,7 @@ void OMR::Power::MemoryReference::accessStaticItem(TR::Node *node, TR::SymbolRef
                cg->addSnippet(snippet);
                }
 
-            TR::MemoryReference *fakeTocRef = new (cg->trHeapMemory()) TR::MemoryReference(cg->getTOCBaseRegister(), 0, sizeof(uintptr_t), cg);
+            TR::MemoryReference *fakeTocRef = new (cg->trHeapMemory()) TR::MemoryReference(cg->getStackPointerRegister(), 0, sizeof(uintptr_t), cg);
```

Comment:
Considering that this is a fake reference whose base register won't actually be used, maybe it would make more sense to use `NULL` here to avoid confusion?

Reply:
Done